typescript/sdk/src/providers/transactions/submitter/ethersV5/schemas.ts

```diff
@@ -7,6 +7,12 @@ export const EV5GnosisSafeTxSubmitterPropsSchema = z.object({
   safeAddress: ZHash,
 });
 
+export const EV5GnosisSafeTxBuilderPropsSchema = z.object({
+  version: z.string(),
```

Comment:
even this imo is not really something that the user should configure

Reply:
Default to 1.0